src/OrchardCore.Modules/OrchardCore.Liquid/Razor/LiquidRazorHelperExtensions.cs

```diff
@@ -0,0 +1,74 @@
+using System.IO;
+using System.Threading.Tasks;
+using Fluid;
+using Microsoft.AspNetCore.Html;
+using Microsoft.Extensions.DependencyInjection;
+using OrchardCore;
+using OrchardCore.Liquid;
+
+public static class LiquidRazorHelperExtensions
+{
+    /// <summary>
+    /// Parses a liquid string to HTML.
+    /// </summary>
+    /// <param name="liquid"></param>
+    /// <returns></returns>
+    public static async Task<IHtmlContent> LiquidToHtml(this IOrchardHelper orchardHelper, string liquid)
+    {
+        return await orchardHelper.LiquidToHtml(liquid, null);
+    }
+
+    /// <summary>
+    /// Parses a liquid string to HTML.
+    /// </summary>
+    /// <param name="liquid">The liquid to parse.</param>
+    /// <param name="model">A model to bind against.</param>
+    /// <summary>
+    public static async Task<IHtmlContent> LiquidToHtml(this IOrchardHelper orchardHelper, string liquid, object model)
```

Comment:
I don't think we need the Raw one. Also this one should specify the HtmlEncoder as in razor we will mostly render html, and the returned `IHtmlContent` assumes that this is already encoded.

Reply:
So I added the Raw one to make it work for https://github.com/OrchardCMS/OrchardCore/issues/3532 so that it could give a normal string back, so that you could then send that string to the markdown extension (which takes a string) `IHtmlContent.ToString()` was giving encoding back and the markdown helper wasn't rendering it correctly. Maybe there is another way for that? 
I will update to specify the HtmlEncoder

Reply:
Hmm, so could just update the markdown helper to be `async` and always render liquid with it, instead of the Raw one, if you prefer?